contentcuration/automation/migrations/0002_auto_20240711_1938.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.2.24 on 2024-07-11 19:38
+from django.db import migrations
+from django.db import models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('automation', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='recommendationscache',
+            name='priority',
+        ),
+        migrations.AddField(
+            model_name='recommendationscache',
+            name='rank',
+            field=models.FloatField(default=0.0, null=True),
+        ),
+    ]
```

Comment:
Instead of having 2+ migrations, because things changed during development, we should just delete and recreate the original migration.